source/docs/software/labView/resources/talon-srx-can.rst

```diff
@@ -0,0 +1,8 @@
+Talon SRX CAN
+=============
+
+The Talon SRX motor controller is a CAN-enabled "smart motor controller" from Cross The Road Electronics/VEX Robotics. The Talon SRX can be controlled over the CAN bus or PWM interface. When using the CAN bus control, this device can take inputs from limit switches and potentiometers, encoders, or similar sensors in order to perform advanced control such as limiting or PID(F) closed loop control on the device.
+
+Extensive documentation about programming the Talon SRX in all three FRC languages can be found in the `Talon SRX Software Reference Manual on CTRE's Talon SRX product page <http://www.ctr-electronics.com/talon-srx.html#product_tabs_technical_resources>`_.
```

Comment:
Convert this to an anonymous hyperlink, guidelines will be updated to reflect this change. 

```suggestion
Extensive documentation about programming the Talon SRX in all three FRC languages can be found in the `Talon SRX Software Reference Manual on CTRE's Talon SRX product page <http://www.ctr-electronics.com/talon-srx.html#product_tabs_technical_resources>`__.
```